instrumentation/wicket-8.0/javaagent/src/test/java/WicketTest.java

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
our convention is to place java tests in the same package as instrumentation code

Reply:
Fixed